src/sql/base/browser/ui/panel/panel.ts

```diff
@@ -266,6 +308,28 @@ export class TabbedPanel extends Disposable {
 		}
 	}
 
+	private focusNextTab(index: number): void {
+		if (index < 0 || index > this.tabList.children.length) {
+			return;
+		}
+		let tab = (<HTMLElement>this.tabList.children[index]);
+		if (tab) {
+			tab.focus();
```

Comment:
This is how focus should be handled. rather than trying to detect a focusable element

Reply:
that just focuses the header... we need inside the body